docs/docs/getting-started/5-nocode-esp32-deployment.md

```diff
@@ -0,0 +1,35 @@
+# No-Code ESP32 Deployment
+
+We built a tool for deploying your Groundlight detector on any supported ESP32 Camera Board without writing any code. This tool is available at https://code.groundlight.ai/groundlight-embedded-uploader/.
+
+## Easy Deployment
+
+This tool is designed to make it as easy as possible to deploy your Groundlight detector on an ESP32 Camera Board. You can deploy your detector in just a few clicks.
+
+1. Go to https://code.groundlight.ai/groundlight-embedded-uploader/.
+2. Plug your ESP32 Camera Board into your computer with a USB cable.
+3. Click through the steps to upload your detector to your ESP32 Camera Board.
+
+## Notification Options
+
+The tool supports the following notification options for your deployed detector:
+
+- Email
+- SMS (With Twilio)
+- Slack
+
+<!-- ## Example Usage
+
+Here is an example of how to use the tool to deploy your detector: -->
+
+## Multiple Supported Boards
+
+The supported boards are:
+
+- M5Stack ESP32 PSRAM Timer Camera X
+- AI-Thinker ESP32-CAM
+- SeeedStudio ESP32S3 Sense
+
+## Source Code
+
+The source code for the arduino binaries is available on GitHub at https://github.com/groundlight/esp32cam-edgelight
```

Comment:
Let's merge the `esp32cam-edgelight` code into the https://github.com/groundlight/esp32cam repo.  They can just be different branches without any shared history.  But having two extremely similar repos is confusing.

Reply:
I didn't know that it's possible to merge one repo into another, how would we do this?